src/index.ts

```diff
@@ -107,6 +111,11 @@ export const whatsappRules: IRule[] = [
     openTag: "<s>",
     wildcard: "~",
   },
+  {
+    closeTag: "</p style='font-family: monospace'>",
+    openTag: "<p>",
+    wildcard: "```",
+  },
```

Comment:
I think the default rules should only include the official Whatsapp ones.

```suggestion
```

Reply:
Hey @flasd, sorry for taking too long to reply.

But whatsapp actually support this wildcard "```" as you can see here: https://faq.whatsapp.com/general/chats/how-to-format-your-messages/?lang=en